ddtrace/contrib/pytest/plugin.py

```diff
@@ -45,15 +51,78 @@ def _store_span(item, span):
     setattr(item, "_datadog_span", span)
 
 
-def _extract_repository_name(repository_url):
-    # type: (str) -> str
-    """Extract repository name from repository url."""
-    try:
-        return compat.parse.urlparse(repository_url).path.rstrip(".git").rpartition("/")[-1]
-    except ValueError:
-        # In case of parsing error, default to repository url
-        log.warning("Repository name cannot be parsed from repository_url: %s", repository_url)
-        return repository_url
+def _extract_reason(call):
+    if call.excinfo is not None:
+        return call.excinfo.value
+
+
+def _get_pytest_command(config):
+    """Extract and re-create pytest session command from pytest config."""
+    command = "pytest"
+    if getattr(config, "invocation_params", None):
+        command += " {}".format(" ".join(config.invocation_params.args))
+    return command
+
+
+def _start_test_module_span(item):
+    """
+    Starts a test module span at the start of a new pytest test package.
+    Note that ``item`` is a ``pytest.Function`` object referencing the test function being run, so we need to go up
+    to find the ``pytest.Package`` ``item``.
+    """
+    test_session_span = _extract_span(item.session)
+    test_module_span = _CIVisibility._instance.tracer._start_span(
+        "pytest.test_module",
+        service=_CIVisibility._instance._service,
+        span_type=SpanTypes.TEST,
```

Comment:
is this the `'type'` tag in the span? This should then be `'test_module_end'`

Reply:
Same as above